### DISCUSSION TITLE
Token list question

### DISCUSSION THREAD

**Discussion by OrionRiftStudios** [bmresearch/Solnet#294] [Q&A] [answered]
Hello I would like to know how to get the available tokens ? 
I know there is a repo for this
https://github.com/solana-labs/token-list

but I would like to know if there is helper or extension to get this information.

Thank you!

**Comment by hoakbuilds** [comment, marked as answer]
We have a `TokenMintResolver` implemented which does this, you can see an example [here](https://github.com/bmresearch/Solnet/blob/master/src/Solnet.Examples/TokenWalletExample.cs#L28), afaik what you're looking for would be something like 
```csharp
var tokenResolver = await TokenMintResolver.LoadAsync();
```

which will load the token list from that json in that repo

**Reply by OrionRiftStudios** [reply]
Thank you
